### PR TITLE
Adds elasticsearch output ssl/auth configuration

### PR DESCRIPTION
**Pull request by marcinc**

### CODE REVIEW

File: run.sh [hunk 1]
```diff
@@ -14,6 +14,11 @@ export HOME=/var/lib/logstash
 
 : ${OUTPUT_ELASTICSEARCH:=true}
 : ${ELASTICSEARCH_HOST:=127.0.0.1:9200}
+: ${ELASTICSEARCH_SSL_ENABLED:=false}
+: ${ELASTICSEARCH_CERTIFICATE_VERIFICATION:=false}
```
Comment (vaijab) [inline run.sh]: Should be `true`.

File: run.sh [hunk 2]
```diff
@@ -14,6 +14,11 @@ export HOME=/var/lib/logstash
...
+: ${ELASTICSEARCH_USER:="none"}
```
Comment (vaijab) [inline run.sh]: What does `none` mean?

**Review by vaijab — CHANGES_REQUESTED** [review]
Please update readme too.

File: run.sh (hunk 2 above)
Comment (marcinc) [reply run.sh]: If left empty strings it'll break template validation. We pass those in deployment so this is only  a placeholder, but I think I change that.

**Comment by marcinc** [comment]
Updated.

**Review by vaijab — APPROVED** [review]
LGTM

**Pull request merged**